src/main/java/com/labs/introtoprogramming/lab4/image/formats/bmp/BMPParser.java

```diff
@@ -0,0 +1,56 @@
+package com.labs.introtoprogramming.lab4.image.formats.bmp;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+
+public class BMPParser {
```

Comment:
Why not move these methods to `BMPImageReader`. To be exact, there is no reason to separate them from the reader.